packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -6489,9 +6489,23 @@ addEventListener('fetch', event => {});`
 		`);
 		});
 
-		it("should output to target es2020 even if tsconfig says otherwise", async () => {
+		it("should output to target es2022 even if tsconfig says otherwise", async () => {
 			writeWranglerToml();
 			writeWorkerSource();
+			fs.writeFileSync(
+				"./index.js",
+				`
+			import { foo } from "./another";
+
+const myArray = ["apple", "banana", "cherry"];
+
+export default {
+  async fetch(request) {
+    return new Response(myArray.at(1));
```

Comment:
Appreciate the suggestion to handle that `.at()` is just passed through regardless of target.

Reply:
Using the top level await, since that is pass through based on the docs you shared AND it will cause a build error on any target build before ES2022.